pkg/cli/mirror/additional.go

```diff
@@ -11,18 +11,17 @@ import (
 	"github.com/openshift/oc/pkg/cli/image/mirror"
 	"github.com/sirupsen/logrus"
 
-	"github.com/RedHatGov/bundle/pkg/cli"
```

Comment:
Is the plan to update the module path to `github.com/openshift/oc-mirror` in a separate PR?

Reply:
Yea I will do this in a separate PR once this PR goes through.